mmcv/ops/csrc/onnxruntime/rotated_feature_align.h

```diff
@@ -0,0 +1,50 @@
+#ifndef ONNXRUNTIME_ROTATED_FEATURE_ALIGN_H
```

Comment:
It is worth to mension that all codes about deployment would be deprecated in the future. You are welcome to contribute the ONNX Runtime ops in [MMDepoy](https://github.com/open-mmlab/mmdeploy).